main.py

```diff
@@ -0,0 +1,66 @@
+from http.client import HTTPException
+
+from fastapi import FastAPI, Depends
+
+import crud
+import schemas
+
+from database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db():
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.post("/authors/", response_model=schemas.Author)
+def create_author(author: schemas.AuthorCreate, db: SessionLocal = Depends(get_db)):
+    return crud.create_author(db=db, author=author)
+
+
+@app.get("/authors/", response_model=list[schemas.Author])
+def read_authors(skip: int = 0, limit: int = 100, db: SessionLocal = Depends(get_db)):
+    authors = crud.get_all_authors(db, skip=skip, limit=limit)
+    return authors
+
+
+@app.get("/authors/{author_id}", response_model=schemas.Author)
```

Comment:
it is better to use the same codestyle as for urls before - add `/` to the end of url